resources/seccomp/aarch64-unknown-linux-musl.json

```diff
@@ -537,6 +524,19 @@
                     }
                 ]
             },
+            {
```

Comment:
Very good catch. However, I'm worried that we had to rely on a person finding this rather than our test. Did we just not cover this in any kind of test on `aarch64`? Maybe we can open an issue about this, might even be a "Good First Contribution" 😄 .

Reply:
We have a regression test for this in `test_api.py::test_map_private_seccomp_regression`, but for some reason this mmap call must not have been triggered on ARM.
It's very likely a platform-specific implementation detail of either the Global Allocator or the rust stdlib.
I'll try to see if sending a larger payload than the current 1 mib would trigger the regression on ARM

Reply:
Turns out that the regression test in `test_api.py` is not failing even if I remove the mmap from the allowlist.
Looking with strace, there are no such mmap calls issued.
I think it's a bug in the test, I'm looking into it